src/components/RockPaperScissors/styles/Scoreboard.styled.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+
+export const StyledIconTitle = styled.img<{ imgSize?: string }>`
+  height: ${(props) => props.imgSize ?? '2rem'};
+  width: ${(props) => props.imgSize ?? '2rem'};
+`
+export const StyledScoreboard = styled.div`
+  width: 100%;
```

Comment:
can consider lowering it to maybe 85% and add margin so there is still some spacing between the scoreboard and the sides of the screen (mobile view affected)